changelogs/fragments/619-Mode-set-for-files-is-applied-to-destination-directory.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- zos_copy - wasn enhanced to work as copy comunitty module for the destitny folder do not change mode 
```

Comment:
Couple of typos in this fragment. I would also describe the work done on zos_copy to be a bugfix, rather than an enhancement. Something like this:
Fixed a bug where the module would change the mode for a directory when copying into it the contents of another.

Reply:
- Ok I will make the changes in the changelogs.
- https://github.com/ansible-collections/ibm_zos_core/issues/417 is already waiting for approval I do not know why I had here the solution too, another change
- And lastly I will verify about copy folder to folder and if it's need just add another verification